models/comments-models.js

```diff
@@ -7,3 +7,16 @@ exports.fetchCommentsByArticleId = async (article_id) => {
   );
   return comments;
 };
+
+exports.addCommentsByArticleId = async (article_id, username, comment) => {
+  const { rows: newComment } = await db.query(
+    `
+  INSERT INTO comments
+  (article_id, author, body)
+  VALUES
+  ($1, $2, $3)
+  RETURNING *;`,
+    [article_id, username, comment]
+  );
+  return newComment;
```

Comment:
Remember that this `newComment` is an array of a single comment at this stage - if you were to return rows[0] for example, it would remove the need to array destructure in the controller